AcadeMeData/migrations/0001_initial.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.9 on 2021-11-23 08:31
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('user',
+                 models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False,
+                                      to='auth.user')),
```

Comment:
I think it should be 
to=settings.AUTH_USER_MODEL,

Reply:
Hi Yariv I think that the 0001_initial.py file is created automaticaly

Reply:
You should review the tool introduction challenge

Reply:
Sorry but I'm confused.. what I've mention is written on page 46 of the tool introduction file..
Can you elaborate?

Reply:
OK, correct, as per the Doc
pipenv run python manage.py makemigrations
correct, i am defining it manually and jusr run migrate, sorry for that